ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -11170,6 +11179,30 @@ static int zend_jit_return(dasm_State **Dst, const zend_op *opline, const zend_o
 
 	if (return_value_used == 0) {
 		|9:
+		if (ZEND_OBSERVER_ENABLED) {
+#if 1
+			|	xor FCARG2a, FCARG2a
+#else
+			if (Z_MODE(op1_addr) == IS_REG) {
+				zend_jit_addr tmp_addr = ZEND_ADDR_MEM_ZVAL(ZREG_R4, TMP_ZVAL_OFFSET);
+
+				if (op1_info & MAY_BE_LONG) {
+					SET_ZVAL_LVAL tmp_addr, Z_REG(op1_addr)
+					SET_ZVAL_TYPE_INFO tmp_addr, IS_LONG
+				} else if (op1_info & MAY_BE_DOUBE) {
```

Comment:
I don't think this branch is taken because of the `#if`, but I imagine `MAY_BE_DOUBE` should be `MAY_BE_DOUBLE`.

Reply:
yeah.

> I don't think this branch is taken because of the `#if`, but I imagine `MAY_BE_DOUBE` should be `MAY_BE_DOUBLE`.

yeah. thanks.